src/frontend/src/flows/register/index.ts

```diff
@@ -37,7 +36,7 @@ import { displayUserNumberWarmup } from "./finish";
 import { savePasskeyOrPin } from "./passkey";
 
 /** Registration (anchor creation) flow for new users */
-export const registerFlow = async <T>({
+export const registerFlow = async ({
```

Comment:
The generic types was used in the showcase and was set to `null`.

But now, I need the connection to be of type AuthenticatedConnection because it's used.

This type was the `T` within the app, but not in the showcase. So, no extra changes were needed in the app part except removing the generic.